### ISSUE TITLE
Vertical vectors not being correctly displayed in proofs.

### ISSUE THREAD

**Issue by 11Kilobytes** [kisonecat/m2o2c2#2]
When I try to enter the following, $\verticalvector{1 \ 2 \ 3}$ into any proof entry textbooks, I get the a horizontal vector as output, instead of a vertical one.

**Comment by StevenGubkin** [comment]
This was a user defined command in our work.  To write matricies in latex
generally, try $\begin{bmatrix} a&b\c&d\e&f \end{bmatrix}$

**Comment by 11Kilobytes** [comment]
Same thing happens.

**Comment by StevenGubkin** [comment]
Ah:  looks like a bug.  One of the \ symbols is being used as an escape in
this format.  Try \\, or \\.  Sorry, hopefully we will have most of the
bugs worked out in the first week.  We do not have play testers except me
and Jim!

**Comment by 11Kilobytes** [comment]
Well, I have been programming for a year now, can I help out with something, perhaps writing tests?
Anyways \\ works.

**Comment by StevenGubkin** [comment]
Yes, please.  We have actually cobbled this whole platform together in
record time with only 3 people really working on it...  It seems to be
working remarkably well considering!  I am not the best person to talk to
about the coding side:  I have been writing most of the content.  Jim,
could you give this guy/gal a heads up about how to contribute?

Peace,
Steve :)
